src/components/new-submission/cz.new-submission-actions.vue

```diff
@@ -5,20 +5,20 @@
       <v-btn v-if="isDevMode" @click="$emit('show-ui-schema')" rounded
         >UI Schema</v-btn
       >
-      <v-btn :id="$id('cancel')" v-if="isEditMode" @click="$emit('cancel')" rounded
+      <v-btn :id="dup_location+'-cancel'" v-if="isEditMode" @click="$emit('cancel')" rounded
         >Cancel</v-btn
       >
       <v-menu :disabled="!errors.length" open-on-hover bottom left offset-y>
         <template v-slot:activator="{ on, attrs}">
           <div v-bind="attrs" v-on="on">
             <v-badge :value="!!errors.length" bordered color="error" icon="mdi-exclamation-thick" overlap>
-              <v-btn :id="$id('save')" @click="$emit('save')" color="primary" :disabled="isSaving || !!errors.length || !hasUnsavedChanges" rounded>
+              <v-btn :id="dup_location+'-save'" @click="$emit('save')" color="primary" :disabled="isSaving || !!errors.length || !hasUnsavedChanges" rounded>
                 {{ isSaving ? "Saving..." : confirmText }}
               </v-btn>
             </v-badge>
 
             <v-badge :value="!!errors.length" bordered color="error" icon="mdi-exclamation-thick" overlap>
-              <v-btn :id="$id('finish')" @click="$emit('save-and-finish')" class="ml-2" color="primary" :disabled="isSaving || !!errors.length" rounded>
+              <v-btn :id="dup_location+'-finish'" @click="$emit('save-and-finish')" class="ml-2" color="primary" :disabled="isSaving || !!errors.length" rounded>
```

Comment:
In this case, instead of passing a property you can simply add a class to this button and use the `id` + the class to select it.

Reply:
hmm are you sure? the class on the buttons is the same other than "ml-2"
`v-btn v-btn--disabled v-btn--has-bg v-btn--rounded theme--light v-size--default`
vs
`ml-2 v-btn v-btn--disabled v-btn--has-bg v-btn--rounded theme--light v-size--default`
doesn't seem too reliable to be selecting elements based on their spacing style classes...

Reply:
@Maurier ^^^^ see comment above

Reply:
but I see your point, instead of passing a PROP, I could make my own distinctive class for each of these buttons and set an ID on the parent component...

Reply:
Exactly, let's just make sure that the distinctive classes we add do not conflict with any helper classes we might have available through https://vuetifyjs.com/